plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -84,6 +84,7 @@ var _ admission.MutationInterface = &PodSecurityPolicyPlugin{}
 var _ admission.ValidationInterface = &PodSecurityPolicyPlugin{}
 var _ genericadmissioninit.WantsAuthorizer = &PodSecurityPolicyPlugin{}
 var _ kubeapiserveradmission.WantsInternalKubeInformerFactory = &PodSecurityPolicyPlugin{}
+var admissionPluginDomain = strings.ToLower(PluginName) + admission.AuditEventAnnotationSuffix
```

Comment:
> nit: and rename the variable to auditKeyPrefix

Done.

Let's name this based on the API rather than the implementation

But not all admission controller has an API. What do you think?

Reply:
Eh, let's deal with it on a case-by-case basis. I'd prefer the annotation reflect the externally facing behavior rather than the implementation.

Reply:
Done.